fabric-biomes-v1/src/main/resources/fabric.mod.json

```diff
@@ -16,8 +16,8 @@
     "FabricMC"
   ],
   "depends": {
-    "fabricloader": ">=0.4.0",
-    "minecraft": ">=1.16-alpha.20.22.a"
+    "fabricloader": ">=0.9.2",
+    "minecraft": ">=1.16.2"
```

Comment:
What version did this biome thing occur in?

.2 is probably not optimistic enough about version ranges, but not a major deal tbf

Reply:
I did not test this with anything lower versions than 1.16.2,  hence the lower bound.

Reply:
This is fine, no need to go lower. 1.16.1 wont work. No need to care about the snapshots or pre-releases here.